packages/terra-spacer/package.json

```diff
@@ -42,7 +42,7 @@
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
     "lint:scss": "stylelint src/**/*.scss",
-    "props-table": "props-table ./src/List.jsx ./src/SelectableList.jsx ./src/SingleSelectList.jsx ./src/MultiSelectList.jsx --out-dir ./docs/props-table",
+    "props-table": "props-table ./src/List.jsx ./src/Spacer.jsx ./src/Spacer.jsx ./src/Spacer.jsx --out-dir ./docs/props-table",
```

Comment:
./src/Spacer.jsx should be listed just once:

`"props-table": "props-table ./src/Spacer.jsx --out-dir ./docs/props-table",`

Reply:
Updated https://github.com/cerner/terra-core/pull/1011/commits/adfd0122d46d706448b6e2064b523f9557267a36